gamestonk_terminal/portfolio/portfolio_helpers.py

```diff
@@ -63,6 +63,23 @@ def rh_positions_to_df(holds: dict):
     return df
 
 
+def dg_positions_to_df(holds):
+    df = pd.DataFrame(columns=["Symbol", "MarketValue", "Quantity", "CostBasis"])
+    sym = []
+    qty = []
+    cb = []
+    for stonk in holds:
+        sym.append(stonk["id"])
+        qty.append(stonk["size"])
+        cb.append(stonk["breakEvenPrice"])
+    df["Symbol"] = sym
+    df["MarketValue"] = 0
```

Comment:
This messes up the formatting a bit. Currently looking at another api I could pull for the marketvalue data